data/distro.yml

```diff
@@ -635,3 +635,60 @@
         <p>Note: for now, graphical installation of Flatpak apps is not possible on Pisi GNU/Linux.</p>
       </li>
     </ol>
+
+- name: "KDE Neon"
+  logo: "kde-neon.svg"
+  info: >
+    <h1>KDE Neon 5.19 and later has Flatpak installed by default.</h1>
+    <p>To activate Flatpak's built-in support just go to Discover > Settings > Flatpak > Add Flathub and mark Flatpak as Default.</p>
+    <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
+    <p></p>
+    <p>In previous versions you must add the URI of the repository to enable Flathub:</p>
+    <ol>
+      <li>
+        <p>Download the <a class="btn btn-default" href="https://flathub.org/repo/flathub.flatpakrepo">Flathub repository file</a>.</p>
+      </li>
+      <li>
+        <p>Go to Discover > Sources > Add Source and select the repository file.</p>
+      </li>
+    <ol>
+
+- name: "OpenMandriva"
+  logo: "openmandriva-blu.svg"
+  info: >
+    <h1>Flatpak and the Discover backend can be installed from Dnfdragora (software manager) starting from OpenMandravia LX 4.</h1>
+    <ol>
+      <li>
+        <p>Download the <a class="btn btn-default" href="https://flathub.org/repo/flathub.flatpakrepo">Flathub repository file</a>.</p>
+      </li>
+      <li>
+        <p>Go to Discover > Sources > Add Source and select the repository file.</p>
+      </li>
+      <li>
+        <p>Then mark Flatpak as Default. It is suggested to restart the system.</p>
+      </li>
+      <li>
+        <p>Now all you have to do is <a href="https://flathub.org/">install some apps</a>!</p>
+      </li>
+    </ol>
+
+- name: "PCLinuxOS"
+  logo: "pclinuxos.svg"
+  info: >
+    <ol class="distrotut">
+      <li>
+        <h2>Install Flatpak</h2>
+        <p>Install the Flatpak package using Synaptic.</p>
```

Comment:
Should perhaps give command line instructions

Reply:
In that case it could be something like:

- Install Flatpak
>su (enter password for root)
>apt-get install flatpak -y

- Add the Flathub repository
>flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo

- it is suggested to reboot the system

----

But the software management in PCLinuxOS is done by Synaptic   #392